Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/README.md

```diff
@@ -0,0 +1,18 @@
+

```

Comment:
Is it possible to add more details? Like https://github.com/colearninglounge/co-learning-lounge/tree/master/Technology/Artificial%20Intelligence